package.json

```diff
@@ -27,6 +27,7 @@
   },
   "dependencies": {
     "cookie": "^0.4.1",
+    "google-libphonenumber": "^3.2.18",
```

Comment:
GDS [recommends](https://design-system.service.gov.uk/patterns/telephone-numbers/) using Google's `libphonenumber` package. 

Note what is imported here is not the official Google npm package (as there isn't one).  I chose [this package](https://www.npmjs.com/package/google-libphonenumber) as it's a straight port of [Google's source code](https://github.com/google/libphonenumber) to npm.  This is the same package chosen by [Virtual Visits](https://github.com/madetech/nhs-virtual-visit/search?q=libphonenumber).

An alternative would have been `libphonenumber-js` ([npm](https://www.npmjs.com/package/libphonenumber-js)) which, although it has more weekly downloads and is lighter weight because it doesn't bundle the Clojure package, is not a straight port of the Google source code and so more likely to have package management issues (IMO).